Cargo.toml

```diff
@@ -59,6 +59,10 @@ uuid = { version = "0.8", features = ["serde", "v4"] }
 async-session = "3.0.0"
 tokio-stream = "0.1.7"
 
+# for oauth example
+oauth2 = "4.1"
+http-types = "2.11.1"
```

Comment:
Is it possible to use the things in `http` or `headers`?

`http-types` provides many of the same things as `http` so including both shouldn't be necessary.

Reply:
I couldn't find similar cookie-building features in `http` or `headers`, but I've replaced http-types with the more purpose-built `cookie` crate, which pulls in less deps.

Reply:
Nice 👍